sormas-app/app/src/main/res/layout/fragment_case_read_layout.xml

```diff
@@ -412,6 +412,40 @@
 
                 </LinearLayout>
 
+                <TextView
```

Comment:
Instead of always showing these fields (including the heading), only show them if the responsible jurisdiction is actually filled in; this also allows you to get rid of the differentJurisdiction ControlTextReadField. Easiest way is probably to wrap all of this information in a layout and use the goneIfEmpty parameter with responsibleRegion (since there should be no way to have a responsible district and community if the region is empty)